tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -304,14 +303,31 @@
       }
 
       .pd-plots-header {
-        height: 50px;
-        min-height: 50px;
-        max-height: 50px;
+        height: 52px;
+        min-height: 52px;
+        max-height: 52px;
+        border-bottom: solid 2px #DADCE0;
       }
 
       .pdplots-holder {
         flex-grow: 1;
-        overflow: auto;
+        overflow-y: overlay;
+        background: #F8F9FA;
+      }
+
+      .pd-range-hyphen {
+        padding-right: 6px;
+        padding-top: 8px;
+      }
+
+      .pd-range-control {
+        border-bottom: solid 1px #FCC934;
+        border-top: none;
+        border-left: none;
+        border-right: none;
```

Comment:
should have border-* none by default. are these necessary?

Reply:
This overrides the default borders on T/R/L and emphasizes the yellow border on the bottom. All other changes for color & grid consistency.